templates/bot/js/workflow/README.md

```diff
@@ -1,103 +1,74 @@
+This template showcases an app that responds to chat commands by displaying UI using an Adaptive Card. The card has a button that demonstrates how to receive user input on the card, do something like call an API, and update the UI of that card. This can be further customized to create richer, more complex sequence of steps which forms a complete workflow.
 
-This application is built with the [Microsoft Bot Framework](https://dev.botframework.com/) running on a restify server running on App Service along with the [Azure Bot Service](https://azure.microsoft.com/services/bot-services/).
+The app template is built using the TeamsFx SDK, which provides a simple set of functions over the Microsoft Bot Framework to implement this scenario.
 
-Here is a screen shot of the application running:
-
-![Responds to command](https://user-images.githubusercontent.com/10163840/192477792-dc447b3a-e304-4cd8-b4df-b1eb9d226292.png)
-
-When you click the `DoStuff` button, the above adaptive card will be updated to a new card as shown below:
-
-![Responds to card action](https://user-images.githubusercontent.com/10163840/192477148-29d9edfc-085b-4d02-b3de-b47b9a456108.png)
-
-# Getting Started
-
-Run your app with local debugging by pressing `F5` in VSCode. Select `Debug (Edge)` or `Debug (Chrome)`.
-
-**Congratulations**! You are running an application that can now send respond to a chat command in Teams.
+## Get Started with the Workflow bot
 
 >
 > **Prerequisites**
 >
-> To run locally, you will need:
+> To run the workflow bot template in your local dev machine, you will need:
 >
 > - `Node.js` installed locally (recommended version: 14)
-> - An [M365 account for development](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts)
+> - An [Microsoft 365 account for development](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts)
 >
 > **Note**
 >
 > Your app can be installed into a team, or a group chat, or as personal app. See [Installation and Uninstallation](https://aka.ms/teamsfx-command-response#customize-installation).
 >
 
-# Understanding the code
+1. First, select the Teams Toolkit icon on the left in the VS Code toolbar.
+2. In the Account section, sign in with your [Microsoft 365 account](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts) if you haven't already.
+3. Press F5 to start debugging which launches your app in Teams using a web browser. Select `Debug (Edge)` or `Debug (Chrome)`.
+4. When Teams launches in the browser, select the Add button in the dialog to install your app to Teams.
+5. Type or select `helloWorld` in the chat to send it to your bot - this is the default command provided by the template.
+6. In the response from the bot, select the **Do Stuff** button.
```

Comment:
**Do Stuff** -> **DoStuff** to keep consistent with the `title` defined in adaptive card.

Reply:
done